cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/ClusterPartitionState.java

```diff
@@ -112,7 +112,7 @@ protected void populateKafkaPartitionState(Set<PartitionInfo> underReplicatedPar
         }
         for (PartitionInfo partitionInfo : _kafkaCluster.partitionsForTopic(topic)) {
           int numInsyncReplicas = partitionInfo.inSyncReplicas().length;
-          boolean isURP = numInsyncReplicas != partitionInfo.replicas().length;
+          boolean isURP = !(partitionInfo.inSyncReplicas() == partitionInfo.replicas());
```

Comment:
I don't think this is safe enough. E.g. if the ISR set is same as replica set, but they are not the same order, then isURP will be true. But I think in that case, isURP should be false.

I feel we should compare the two Node[] in a sorted manner